api4/user.go

```diff
@@ -88,6 +90,34 @@ func getUser(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getUserByEmail(c *Context, w http.ResponseWriter, r *http.Request) {
+
+	if c.Err != nil {
```

Comment:
You need to do a `c.RequireEmail()` (you'll need to add that function in context.go) check right above this otherwise `c.Err` won't get set